build_projects/dotnet-host-build/sign.proj

```diff
@@ -35,6 +35,9 @@
       <FilesToSign Include="$(OutDir)corehost/**/hostpolicy.dll">
         <Authenticode>$(CertificateId)</Authenticode>
       </FilesToSign>
+      <FilesToSign Include="$(OutDir)corehost/**/dotnet.exe">
```

Comment:
That looks good.  If you want to confirm you can run the same commands as the official build:
`build_projects\dotnet-host-build\build.ps1 -Configuration Release -Targets Prepare,CompileCoreHost`, then
`msbuild build_projects\dotnet-host-build\sign.proj /t:SignCoreHost /p:Configuration=Release /p:RID=win81-x86`
and look for the "Fake sign target.  Would sign:" message on your local build to make sure it includes dotnet.exe.  I don't know if this applies, but since we were just looking at this in CoreFX, this is Authenticode only, you need a \<StrongName\> line if you want that as well.